wgpu/resources/codegen_report.md

```diff
@@ -20,7 +20,7 @@
 * Diffs for GPUQueue: add read_buffer, add read_texture, hide copy_external_image_to_texture
 * Validated 37 classes, 112 methods, 45 properties
 ### Patching API for backends/wgpu_native/_api.py
-* Validated 37 classes, 114 methods, 0 properties
+* Validated 37 classes, 98 methods, 0 properties
```

Comment:
For the record (I had to check this), the reduced checked methods are the Python `_release()` methods that have been removed. The C-calls are still checked. The `Validated 131 C function calls` is unchanged.